Fix #95

vtgav opened the pull request:
updated doc / pdf to 3.1, added a ppt for the lifecycle diagram, updated the schema

vtgav committed:
Fix #95

vtgav committed:
Fix #95 - Update README.md

vtgav committed:
Fix #95 - Delete Lucid protocol specification v3.0.docx

vtgav committed:
Fix #95 - Delete Lucid protocol specification v3.0.pdf

vtgav committed:
Fix #95

scombellack reviewed (changes requested):
My reading of the spec looks like the "cVer" field should be in the data message header (sent only once per data message), alongside 'p'.  The current change has the cVer inside the 'p' array objects.
Section 7, Figure 12 on page 71 should have "Event: /UP/data" moved to the "( if consistent versions )" list, shouldn't it?

vtgav committed:
Delete Lucid protocol diagrams v3.1.pptx

Stu noticed an error on this diagram

vtgav committed:
Delete Lucid protocol specification v3.1.docx

Stu noticed an error on the level of cVer in the data JSON

vtgav committed:
Delete Lucid protocol specification v3.1.pdf

removed as doc file is being replaced

vtgav committed:
Update lucid.schema.json

added cVer to the required list for the data message

vtgav committed:
Add files via upload

new versions reflecting errors spotted by Stu
1. cVer moved to top level in data message
2. lifecycle diagram corrected
3. cVer added to required list for data message in schema

vtgav commented:
updated doc/pdf, lifecycle diagram and schema in line with Stu's comment

scombellack reviewed (approved):
Yes, changes seem to be correct
